cluster/clean.sh

```diff
@@ -45,6 +45,9 @@ for i in ${namespaces[@]}; do
     _kubectl -n ${i} delete clusterrolebinding -l 'kubevirt.io'
     _kubectl -n ${i} delete clusterroles -l 'kubevirt.io'
     _kubectl -n ${i} delete serviceaccounts -l 'kubevirt.io'
+    if [ $(_kubectl -n ${i} get crd offlinevirtualmachines.kubevirt.io | wc -l) -gt 0 ]; then
```

Comment:
Could you add a FIXME to remove it in a few days add here? It is just a workaround ...